.github/ISSUE_TEMPLATE/4-constraint-work.yaml

```
@@ -0,0 +1,112 @@
+name: Create or Revise a Constraint
+description: Define work to be performed specifically related to the creation, refinement, or decommissioning of constraints.
```

Minor: can we use plainer language?

```suggestion
description: Define work to be performed specifically related to the adding, changing, or removing constraints.
```